packages/driver/src/multi-domain/cypress.ts

```diff
@@ -30,6 +32,18 @@ const createCypress = () => {
     setup(config, env)
   })
 
+  Cypress.specBridgeCommunicator.on('generate:final:snapshot', (snapshotUrl: string) => {
+    const specBridgeLocation = $Location.create(window.location.href)
```

Comment:
Are we able to reference cy.state('autOrigin') here?

Reply:
updated in [c264f20](https://github.com/cypress-io/cypress/pull/20949/commits/c264f206d6ffcdfec536f4e2800f20b9a59bfcf6)